### PR TITLE
Make sure PRs that include `## :warning: BREAKING` in their body have the `kind/breaking` label

### PR DESCRIPTION
**Pull request by yardenshoham**
Just a small addition because this can be automated

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ The script will also look for merged pull requests that have the labels
 It will also search for pull requests that target release branches and remove
 any `backport/*` labels from them.
 
+It will also make sure any pull requests that have
+`## :warning: BREAKING :warning:` in their description have the `kind/breaking`
+label.
```
Comment (delvh) [inline README.md:30]: Also the other way around?

File: src/github.ts [hunk 2]
```diff
@@ -81,6 +81,19 @@ export const fetchUnmergedClosedWithMilestone = async (
   return json;
 };
 
+// returns a list of breaking PRs that don't have the label kind/breaking
+export const fetchBreakingWithoutLabel = async () => {
+  const response = await fetch(
+    `${GITHUB_API}/search/issues?q=` +
+      encodeURIComponent(
+        `is:pr "## :warning: BREAKING :warning:" -label:kind/breaking repo:go-gitea/gitea`,
```
Comment (delvh) [inline src/github.ts:89]:
```suggestion
        `is:pr "## :warning: BREAKING" -label:kind/breaking repo:go-gitea/gitea`,
```
I think that prefix suffices as well.

File: README.md (hunk 1 above)
Comment (yardenshoham) [reply README.md:30]: What do you mean?

Comment (delvh) [reply README.md]: `It will make sure that any pull request with the `kind/breaking` label have
`## :warning: BREAKING :warning:` in their description.`

Comment (yardenshoham) [reply README.md]: With a check similar to `giteabot/lgtm`?

Comment (delvh) [reply README.md]: Yes.

**Review by delvh — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (yardenshoham) [reply README.md]: Interesting. Another option would also be to remove `kind/breaking` if ``## :warning: BREAKING`` is not in the description automatically, like we do on PRs targeting release branches (we remove backport labels from them)

Comment (delvh) [reply README.md]: Hmm. I don't think it's a good idea to remove a label that is intended as protection.
The backport labels have a functional purpose that doesn't apply under the given circumstances.
But a PR is not not-breaking given that the disclaimer is missing.

**Comment by yardenshoham** [comment]
It works!
image

**Pull request merged**